src/index.html

```diff
@@ -11,7 +11,92 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <h1 class="h1form">HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post"
+      onsubmit="onSubmit()">
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+        <label for="surname">Surname:
+          <input type="text" name="surname"
+            id="surname">
```

Comment:
```suggestion
        <label for="surname">
          Surname:
          <input
            type="text"
            name="surname"
            id="surname"
          >
```
Форматируй теги, в которых больше 2 атрибутов, таким образом